README.md

```diff
@@ -85,35 +85,48 @@ Comments:
 - 'y = y[t-1] * rog' is formula (equation)
 
 Programming entry point
------------------------
+=======================
 ```python
-from xl import XlSheet
+from xlsmodel.xl import XlSheet
 XlSheet(filename="test1.xls", sheet="input_sheet_v1", anchor="A1").save(sheet="output_v2")
 ```
 
-Command line call 
------------------
-```
-python xl.py <filname> <sheet> <anchor>
-   <filename>: filename ot path to .xls file
-   <sheet>: string representing sheet name or integer based at 1, defaults to first sheet in file  
-   <anchor>: string with A1 style reference pointing to start of datablock on sheet, defaults to "A1"
-
-python xl.py test0.xls
-python xl.py test0.xls 1 A1
-python xl.py test0.xls sheet1 A1
+Command line interface
+======================
+```bash
+# TODO(dmu) LOW: Is this usage really needed here? We can get it any time with:
+# python -m xlsmodel.xl --help
+usage: python -m xlsmodel.xl [-h] filename [sheet] [anchor]
```

Comment:
(1) xlsmodel.xl now looks a bit confusing - as if xl is file extension, may need to rename xl.py into something else
(2) will xlsmodel.xl filename [sheet] [anchor] [-h] be any better?